Style fixes to the main page

lucasmbispo opened the pull request:
- Fixed space between some elements
![image](https://user-images.githubusercontent.com/101370137/229911835-d9aea7c8-519e-4dc1-b216-8d6ebb9012eb.png)
- Fixed 'Docs' index page titles hover behavior
![image](https://user-images.githubusercontent.com/101370137/229912107-8c200242-e64d-4025-8d0f-0eea6efdbe28.png)
- Fixed typo in Contact CTA
![image](https://user-images.githubusercontent.com/101370137/229912215-e4a8fd2c-c929-4c76-847f-eeb0366405ff.png)
- Fixed footer links alignment
![image](https://user-images.githubusercontent.com/101370137/229912407-b1c943f3-6e10-4b30-8e74-a003749cc265.png)
- Added Animated Icons
![image](https://user-images.githubusercontent.com/101370137/229912607-faadf8da-3727-4add-b297-37ab72aa7028.png)

popovayoana commented:
@lucasmbispo great, just one request, can we change the bitcoin icon for "cost-effective" with this one: https://www.flaticon.com/free-animated-icon/ewallet_7994345?term=money&page=1&position=22&origin=search&related_id=7994345

lucasmbispo commented:
[Preview](https://04c92a91.datahub-next.pages.dev/)

@popovayoana 
- Added more spacing for 'Features' elements breathe a little more.
- Added animated icons for 'Sign Up'.

@anuveyatsu do we need the 'Edit this page' button on 'datahub.io'? I've hidden as it was returning 'undefined' object on click. If we do need it, it's easily reversible.

rufuspollock commented:
@lucasmbispo this has conflicts. Can you fix and we can merge.
